example/src/Components/SettingsTable/index.tsx

```diff
@@ -66,8 +68,12 @@ const SettingsTable: FC<SettingsTableProps> = ({ transformationFunction }) => {
     }
   };
 
-  const handleSettingsChange = (path: string, value: boolean | string) => {
-    let updatedValue = { ..._.set(settings, path, value) };
+  const handleSettingsChange = async (
+    path: string,
+    value: boolean | string
+  ) => {
+    const currentSettings = settings ?? JSON.parse(await getItem());
+    let updatedValue = { ..._.set(currentSettings, path, value) };
```

Comment:
In the case of an empty or null JSON, this will cause `undefined` issues,
I suspect these issues can be reproduced when you remove app data and restart the sample app, maybe worth testing

Reply:
I am not sure if I understood your comment correctly Stefan. If there is any other situation you meant that may cause problems, let me know, I am happy to test.